### DISCUSSION TITLE
how can change the logs duration?

### DISCUSSION THREAD

**Discussion by jotaah123** [telekom-security/tpotce#1023] [Q&A] [answered]
Hello dears , 
in some web I saw that tpot erase the logs at 1 month, I need change the logs duration to 3 or 6 month saved on the server ,after that clean's the logs 
please help me :(

**Comment by t3chn0m4g3** [comment, marked as answer]
It's 90 days by [default](https://github.com/telekom-security/tpotce/blob/cac7cdcec6288a92703880c87fb0a699b71cf283/etc/curator/actions.yml#L26). Just adjust the file to your needs in `/opt/tpot/etc/curator`.

**Reply by jotaah123** [reply]
Thanks so much for the answer @t3chn0m4g3 , i edited the `action.yml` and change the ` unit_count: 90 ` to `unit_count: 180 `
:)

**Comment by t3chn0m4g3** [comment]
Welcome.
If it is the honeypot logs you want to keep longer, you need to adjust [this](https://github.com/telekom-security/tpotce/blob/master/etc/logrotate/logrotate.conf) from 30 to what you need in `/opt/tpot/etc/logrotate/`.
